src/Components/Samples/BlazorUnitedApp/App.razor

```diff
@@ -9,33 +9,10 @@
     <link rel="stylesheet" href="css/site.css" />
     <link href="BlazorUnitedApp.styles.css" rel="stylesheet" />
     <link rel="icon" type="image/png" href="favicon.png" />
-    <HeadOutlet></HeadOutlet>
+    <HeadOutlet />
 </head>
 <body>
-    <Router AppAssembly="@typeof(App).Assembly">
-        <Found Context="routeData">
-            <RouteView RouteData="@routeData" DefaultLayout="@typeof(MainLayout)" />
-            <FocusOnNavigate RouteData="@routeData" Selector="h1" />
-        </Found>
-        <NotFound>
-            <PageTitle>Not found</PageTitle>
-            <LayoutView Layout="@typeof(MainLayout)">
-                <p role="alert">Sorry, there's nothing at this address.</p>
-            </LayoutView>
-        </NotFound>
-    </Router>
-
-    <div id="blazor-error-ui">
-        <environment include="Staging,Production">
-            An error has occurred. This application may no longer respond until reloaded.
-        </environment>
-        <environment include="Development">
-            An unhandled exception has occurred. See browser dev tools for details.
-        </environment>
-        <a href="" class="reload">Reload</a>
-        <a class="dismiss">🗙</a>
-    </div>
-
+    <Routes />
```

Comment:
The reason for changing `Samples/BlazorUnitedApp` like this is just to bring it more into alignment with conventional projects.

What we had here before was a pretty weird point-in-time snapshot from the early .NET 8 days and couldn't easily be switched between global and per-page interactivity, which is something you might often want to do when experimenting with different features.